.gitlab/issue_templates/Bug Report.md

```diff
@@ -1,30 +0,0 @@
-## Summary
```

Comment:
In theory, we could still keep the GitLab-related files and also move the `.gitlab.ci-yml` file to the `.gitlab` folder. Removing the files makes the project easier to maintain, though, so I'd be fine with getting rid of the GitLab stuff.

Reply:
Yes, I think removing them should be fine, especially when keeping in mind that the files aren't lost, since they're still in the commit history.

Reply:
That's true ;)